backend/src/test/java/com/woowacourse/ternoko/service/ReservationServiceTest.java

```diff
@@ -172,10 +190,194 @@ void createReservationException() {
+
+        final Reservation reservation = reservationService.create(CREW1.getId(),
+                new ReservationRequest(COACH3.getId(), LocalDateTime.of(NOW_PLUS_2_DAYS, FIRST_TIME),
+                        FORM_ITEM_REQUESTS));
+        // when
+        assertThatThrownBy(() -> reservationService.update(CREW2.getId(), reservation.getId(),
+                new ReservationRequest(COACH3.getId(), LocalDateTime.of(NOW_PLUS_3_DAYS, SECOND_TIME),
+                        FORM_ITEM_UPDATE_REQUESTS)))
+                .isInstanceOf(InvalidReservationCrewIdException.class)
+                .hasMessage(ExceptionType.INVALID_RESERVATION_CREW_ID.getMessage());
+    }
+
+    @Test
+    @DisplayName("면담 예약을 수정 시 선택한 코치가 존재하지 않는다면 예외를 반환한다.")
+    void update_WhenCoachNotFound() {
+        // given
+        coachService.putAvailableDateTimesByCoachId(COACH3.getId(), MONTH_REQUEST);
+
+        final Reservation reservation = reservationService.create(CREW1.getId(),
+                new ReservationRequest(COACH3.getId(), LocalDateTime.of(NOW_PLUS_2_DAYS, FIRST_TIME),
+                        FORM_ITEM_REQUESTS));
+        // when
+        assertThatThrownBy(() -> reservationService.update(CREW1.getId(), reservation.getId(),
+                new ReservationRequest(5L, LocalDateTime.of(NOW_PLUS_3_DAYS, SECOND_TIME),
+                        FORM_ITEM_UPDATE_REQUESTS)))
+                .isInstanceOf(CoachNotFoundException.class)
+                .hasMessage(5L + ExceptionType.COACH_NOT_FOUND.getMessage());
+    }
+
+    @Test
+    @DisplayName("면담 예약 수정 시 코치의 가능 시간이 아니라면 예외를 반환한다.")
+    void update_WhenInvalidAvailableDateTime() {
+        // given
+        coachService.putAvailableDateTimesByCoachId(COACH3.getId(), MONTH_REQUEST);
+
+        final Reservation reservation = reservationService.create(CREW1.getId(),
+                new ReservationRequest(COACH3.getId(), LocalDateTime.of(NOW_PLUS_2_DAYS, FIRST_TIME),
+                        FORM_ITEM_REQUESTS));
+        // when & then
+        assertThatThrownBy(() -> reservationService.update(CREW1.getId(), reservation.getId(),
+                new ReservationRequest(COACH3.getId(), LocalDateTime.of(NOW_PLUS_1_MONTH, SECOND_TIME),
+                        FORM_ITEM_UPDATE_REQUESTS)))
+                .isInstanceOf(InvalidReservationDateException.class)
+                .hasMessage(ExceptionType.INVALID_AVAILABLE_DATE_TIME.getMessage());
+    }
+
+    @Test
+    @DisplayName("면담 수정 시, 당일 예약을 시도하면 에러가 발생한다.")
+    void updateReservationTodayException() {
+        // given
+        coachService.putAvailableDateTimesByCoachId(COACH4.getId(), MONTH_REQUEST);
+
+        final Reservation reservation = reservationService.create(CREW1.getId(),
+                new ReservationRequest(COACH4.getId(), LocalDateTime.of(NOW_PLUS_2_DAYS, FIRST_TIME),
+                        FORM_ITEM_REQUESTS));
+        // when & then
+        coachService.putAvailableDateTimesByCoachId(COACH4.getId(), PAST_REQUEST);
```

Comment:
`put` 하는 부분이 `given`으로 들어가는 것이 좋아보여용
```suggestion
        final Reservation reservation = reservationService.create(CREW1.getId(),
                new ReservationRequest(COACH4.getId(), LocalDateTime.of(NOW_PLUS_2_DAYS, FIRST_TIME),
                        FORM_ITEM_REQUESTS));
                        coachService.putAvailableDateTimesByCoachId(COACH4.getId(), PAST_REQUEST);
        // when & then
        
```

Reply:
처음에 말씀해주신대로 put()을 given에 넣어두었는데, 되는 시간이 PUT이다 보니 같은 달일 경우 기존 데이터(MONTH_REQUEST)가  사라질 수도 있어 예약 시점 이후에 하도록 작성해보았습니다.
해당 내용은 추후에 픽스처에 대해 리팩터링 후 진행하도록 하겠습니다.